package.json

```diff
@@ -9,15 +9,23 @@
     "@anthropic-ai/tokenizer": "^0.0.4",
     "@inquirer/prompts": "^4.3.0",
     "cross-spawn": "^7.0.3",
+    "directory-tree": "^3.5.1",
+    "mime-types": "^2.1.35",
+    "next": "^14.1.4",
+    "nextra": "^2.13.4",
+    "nextra-theme-docs": "^2.13.4",
+    "react": "^18.2.0",
```

Comment:
We accidentally added nextra and nextra-theme-docs into Lumentis' dependencies :)

Reply:
Removed